timelock-impl/src/main/java/com/palantir/atlasdb/timelock/adjudicate/ServiceLevelObjectiveSpecification.java

```diff
@@ -27,6 +27,7 @@
     Duration maximumPermittedP99();
     double maximumPermittedErrorProportion();
     double minimumRequestRateForConsideration();
+    double p99Multiplier();
```

Comment:
I'd prefer a design where we have something like

```
  Duration maximumPermittedSteadyStateP99();
  Duration maximumPermittedQuietP99(); // currently this is maximumPermittedP99 * multiplier
```

with comments explaining the difference. The multiplier doesn't really say what it does